packages/viewer/tests/cypress/support/commands.ts

```diff
@@ -282,10 +283,17 @@ Cypress.Commands.add('clickOnLanguage', (lang) => {
 })
 
 Cypress.Commands.add('waitUntilState', (predicate, options) => {
+    /**
+     * Full explanation of the ts expect error here :
+     * AUTWindow's type includes the classic window type, as well as an `Application Window` type
+     * which is used to store custom arguments. The issue is that those arguments are not
+     * known in advance, and typescript doesn't like that.
+     */
     cy.waitUntil(
         () =>
             cy
                 .window({ log: false })
+                // @ts-expect-error Application window definition has no defined keys in its definition
```

Comment:
there's already some type definition for that somewhere (I think it's in `src/env.d.ts`), you could add that type definition to our main `cypress.d.ts`